packages/app/components/Queue/Student/StudentQueue.tsx

```diff
@@ -292,7 +296,13 @@ export default function StudentQueue({
             buttons={
               !studentQuestion && (
                 <Popconfirm
-                  title="In order to join this queue, you must delete your previous question. Do you want to continue?"
+                  title={
+                    <PopConfirmTitle>
+                      You already have a question in a queue for this course, so
```

Comment:
So we allow them to have multiple questions across the different courses. Do we still need this popup? 

Can't we just disable the button altogether? if they have a question in this course, then they can just edit their existing question. 

I think we can Disable the button and add a tooltip indicating that they can only have 1 question in the queue at a time

Reply:
Oh so we were thinking about hybrid queues when we implemented this.
Maybe there are students who want to leave an in-person queue for whatever reason and join an online queue instead (or vice versa)?

Reply:
Yea that makes sense, we can keep it in then